README.md

```diff
@@ -80,3 +80,13 @@ You will need to set the following environment variables on the frontend:
 ### Development
 
 Click `Login` to log in as a test user. The `./manage.py populate` command creates a test user for you with username `bfranklin` and password `test`. Go to `/api/admin` to login to this account.
+
+#### Ticketing
+
+To test ticketing locally, you will need to [install](https://github.com/FiloSottile/mkcert?tab=readme-ov-file#installation) `mkcert`, enter the `frontend` directory, and run the following commands:
+
+- `$ mkcert -install`
+- `$ mkcert localhost 127.0.0.1 ::1`
+- `$ export DOMAIN=https://localhost NODE_TLS_REJECT_UNAUTHORIZED=0`
+
+Then, after the frontend is running, run `docker-compose up` **in a new terminal window** and access the application at [https://localhost](https://localhost).
```

Comment:
Thanks for the writeup. Might be a dumb question, but what happens if someone doesn't follow these instructions? I'm guessing the Cybersource ticketing routes will still work, but the capture context it generates won't be displayable on the frontend.

Would be ideal if there was a way to do dev work on non-ticketing projects without setting up the HTTPS/nginx bridge.

Reply:
Yep, the iframe will not be rendered on the frontend without HTTPS but the backend routes will still work. The dev workflow for non-ticketing tasks should be unaffected by this change